manual/src/foreword.etex

```diff
@@ -54,13 +54,24 @@ Automatique (INRIA).
 The OCaml documentation and user's manual is licensed under a Creative
 Commons Attribution-ShareAlike 4.0 International License (CC BY-SA
 4.0), \url{https://creativecommons.org/licenses/by-sa/4.0/}.
+
+The sample code in the user's manual and in the reference documentation
+of the standard library is licensed under a Creative Commons
+CC0 1.0 Universal (CC0 1.0) Public Domain Dedication License,
+\url{https://creativecommons.org/publicdomain/zero/1.0/}.
 \end{latexonly}
 
 \begin{htmlonly}
 \begin{rawhtml}
-<a id="cc_license_logo" rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png"></a>
+<a id="cc_license_logo" rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png"></a>
```

Comment:
Note the URL was changed here because that's where it redirects.

Reply:
Is it a good idea still? The first URL looks more official-looking and may be more stable? (No strong opinion.)

Reply:
Yes, it's a 301. That means update your links.

```
> curl -s -o /dev/null -w "%{http_code}" https://i.creativecommons.org/l/by-sa/4.0/88x31.png
301
```